lms/static/lms/js/xblock/lms.runtime.v1.js

```diff
@@ -0,0 +1,55 @@
+(function(URI) {
+    'use strict';
+
+    var __hasProp = {}.hasOwnProperty,
+        __extends = function(child, parent) {
+            var key;
+            for (key in parent) {
+                if (__hasProp.call(parent, key)) {
+                    child[key] = parent[key];
+                }
+            }
+            function Ctor() {
+                this.constructor = child;
+            }
+            Ctor.prototype = parent.prototype;
+            child.prototype = new Ctor();
+            child.__super__ = parent.prototype;
+            return child;
+        };
```

Comment:
It is added when in Coffee you use the ``extends`` keyword, which in this case was:

    class LmsRuntime.v1 extends XBlock.Runtime.v1

I'm fairly certain we need it (or something equivalent) to implement inheritance the way the code expects. We discussed putting the function into a shared place but decided not to for this go around.